### PR TITLE
Bugfix/ical minor bugs

### PR DESCRIPTION
**Pull request by 4D44H**
Fixes for:
- https://trello.com/c/Pa8230u4/233-1-ical-personal-feed-mixed-up-event-time-and-shift-time-in-event-description
- https://trello.com/c/1hUZ7BU5/237-1-ical-mobile-reminder-field-has-no-line-breaks
- https://trello.com/c/Q0nzTjhp/224-1-event-view-person-who-created-an-event-can-see-the-un-publish-button-even-if-they-have-no-rights-to-use-it

### CODE REVIEW

File: app/Http/Controllers/IcalController.php [hunk 1]
```diff
@@ -264,15 +268,17 @@ public function userScheduleWithAlarm($prsn_uid, $alarm = 0)
                     $vEvent->setSummary("".($schedule->event->evnt_title)." - ".($evt->jobType->jbtyp_title));
                     
                     $eventLink = "".URL::route('event.show', $schedule->event->id);
-                    $eventTime = $evt->entry_time_start;
+                    $eventTimeStart = $schedule->event->evnt_time_start;
```
Comment (P1tt187) [inline app/Http/Controllers/IcalController.php]: the event is now not respecting the individual time of a shift NOK

Comment (P1tt187) [inline app/Http/Controllers/IcalController.php]: my fault, just a bug in my calendar software, looks good OK

File: app/Http/Controllers/IcalController.php [hunk 2]
```diff
@@ -353,8 +353,8 @@ public function singleEvent($evt_id)
                 $vEvent->setSummary($event->evnt_title);
                 
                 $eventLink = "".URL::route('event.show', $event->id);
-                $eventTimeStart = $event->evnt_time_start;
-                $eventTimeEnd = $event->evnt_time_end;
+                $eventTimeStart = substr('H:i', $event->evnt_time_start, 0, 5);
```
Comment (P1tt187) [inline app/Http/Controllers/IcalController.php]: substr accepting only 3 arguments, causes exception NOK

Comment (4D44H) [inline app/Http/Controllers/IcalController.php]: Forgot to remove "H:i", my bad. Also on 357.

**Comment by 4D44H** [comment]
@P1tt187 fixed. Any other changes?

**Pull request merged**